runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -230,15 +232,19 @@ public void write(IToken token, Object data) throws OverwriteException {
                     .peek(e, address));
     }
 
+    public @NonNull ILogData read(long address) {
```

Comment:
Please add comments for the new public api

Reply:
Done.